artiq/coredevice/phaser.py

```diff
@@ -1065,35 +1065,34 @@ def set_servo(self, bypass=1, hold=0, profile=0):
         :param hold: 1 to hold the servo IIR filter output constant, 0 for normal operation
         :param profile: profile index to select for channel (0 to 3)
         """
-        if (profile < 0) | (profile > 3):
+        if (profile < 0) or (profile > 3):
             raise ValueError("invalid profile index")
         addr = PHASER_ADDR_SERVO_CFG0 + self.index
         if bypass == 0:
             data = 1
         if hold == 1:
-            data = data | (1 << 1)
+            data = data or (1 << 1)
```

Comment:
No. These should obviously be bitwise operation.

Reply:
whops, thanks.

Reply:
There's more stuff wrong here